Update doc engine and fix edit links (#249)
mirumee/saleor-docs: README.md

1. Start Saleor API at `http://localhost:8000`
1. Run `npm run update-api-reference`

# Debugging

> In dev mode Docusaurus serve debug page with list of all available routes and config at http://localhost:3000/__docusaurus/debug.

Reviewer: In dev mode, Docusaurus is used for debugging a page with a list of all available routes and config at http://localhost:3000/__docusaurus/debug.

> In dev mode Docusaurus serve debug page with list of all available routes and config at http://localhost:3000/__docusaurus/debug.

Reviewer: ```suggestion
In dev mode, Docusaurus is used for debugging a page with a list of all available routes and config at http://localhost:3000/__docusaurus/debug.
```
